docs/sources/alerting/monitor/meta-monitoring/_index.md

```diff
@@ -2,7 +2,8 @@
 aliases:
   - meta-monitoring/
   - alerting/meta-monitoring/
-canonical: https://grafana.com/docs/grafana/latest/alerting/set-up/meta-monitoring/
+  - /alerting/set-up/meta-monitoring/
```

Comment:
I believe the intent of this alias is to redirect the old location of https://grafana.com/docs/grafana/latest/alerting/set-up/meta-monitoring/ to https://grafana.com/docs/grafana/latest/alerting/monitor/meta-monitoring/.

The alias to do that is:

```suggestion
  - ../set-up/meta-monitoring/
```